docs/theming.md

```diff
@@ -156,7 +156,7 @@ import React from 'react';
 import kind from '@enact/core/kind';
 import UiButton from '@enact/ui/Button';
 
-import componentCss from './Button.less';
+import componentCss from './Button.module.less';
```

Comment:
Nice catch on the `.module.less`.  Do we have any of that in any of the tutorials that we need to update?

Reply:
Probably!  I will take a quick look.

Reply:
I only found one more example in the Enact migration docs, and it's updated now.